### DISCUSSION TITLE
Weblate create empty component via rest api

### DISCUSSION THREAD

**Discussion by mateusz-olszak** [WeblateOrg/weblate#8282] [General]
Hi,
I would like to create a new component via rest api and have two question for that:
1. I want to know if there is any possibility to create an empty component from scratch via rest api as we can do using UI panel.
2. If there is an option to create initialized component with some translation using json. I mean I can achieve this uploading json file as some multipart but I'd like to know if I can initialize this with json data omitting createing some additional json file.

**Comment by nijel** [comment]
There should be no issue in using local VCS in https://docs.weblate.org/en/latest/api.html#post--api-projects-(string-project)-components-
